next-sitemap.config.js

```diff
@@ -0,0 +1,18 @@
+// next-sitemap.config.js
+
+/** @type {import('next-sitemap').IConfig} */
```

Comment:
Why this commented out lines?

Reply:
This is a JSDoc comment that provides type information for the module.exports object.